packages/test-runner-commands/browser/commands.mjs

```diff
@@ -61,6 +61,10 @@ export function emulateMedia(media) {
   return executeServerCommand('emulate-media', media);
 }
 
+export function emulateHover(options) {
```

Comment:
I would suggest this to `hover` because we don't **emulate** it (unlike media query command) but trigger the browser API to actually move cursor over the element so it gets `:hover` state.

Reply:
Good point, I'll rename it.